modules/analytics/src/views/full/index.tsx

```diff
@@ -126,6 +129,13 @@ const fetchReducer = (state: State, action): State => {
       dateRange,
       disableAnalyticsFetching: true
     }
+  } else if (action.type === 'receivedQnaQuestions') {
+    const { qnaQuestions } = action.data
+
+    return {
+      ...state,
+      qnaQuestions: { ...qnaQuestions }
```

Comment:
`qnaQuestions: action.data.qnaQuestions`

Reply:
Done in 13eb5c6